WordPress/src/main/java/org/wordpress/android/ui/media/MediaBrowserActivity.java

```diff
@@ -282,9 +282,13 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
         switch (requestCode) {
             case RequestCodes.PICTURE_LIBRARY:
             case RequestCodes.VIDEO_LIBRARY:
-                Uri imageUri = data.getData();
-                String mimeType = getContentResolver().getType(imageUri);
-                fetchMedia(imageUri, mimeType);
+                if (resultCode == Activity.RESULT_OK) {
```

Comment:
This is perhaps picky, but we can simplify this to `if (resultCode == Activity.RESULT_OK && data != null)`.